packages/firestore/src/local/indexeddb_query_cache.ts

```diff
@@ -97,36 +97,79 @@ export class IndexedDbQueryCache implements QueryCache {
     );
   }
 
+  private saveMetadata(
```

Comment:
Move this down with the other private helper methods? (probably next to saveQueryData)

Reply:
Done.